src/main/client_context_file_opener.cpp

```diff
@@ -22,4 +27,15 @@ bool FileOpener::TryGetCurrentSetting(FileOpener *opener, const string &key, Val
 	return opener->TryGetCurrentSetting(key, result);
 }
 
+bool FileOpener::TryGetCurrentSetting(FileOpener *opener, const string &key, Value &result, FileOpenerInfo &info) {
+	if (!opener) {
+		return false;
+	}
+	return opener->TryGetCurrentSetting(key, result, info);
+}
+
+bool FileOpener::TryGetCurrentSetting(const string &key, Value &result, FileOpenerInfo &info) {
+	return false;
```

Comment:
Should this default to calling the none FileOpenerInfo method i.e.
```cpp
bool FileOpener::TryGetCurrentSetting(const string &key, Value &result, FileOpenerInfo &info) {
	return this->TryGetCurrentSetting(key, value);
}
```

Reply:
Yes. Corrected.